virttest/utils_test/libvirt.py

```diff
@@ -507,76 +507,6 @@ def setup_or_cleanup_iscsi(is_setup, is_login=True,
     return ""
 
 
-def get_host_ipv4_addr():
-    """
-    Get host ipv4 addr
-    """
-    if_up = utils_net.get_net_if(state="UP")
-    for i in if_up:
-        ipv4_value = utils_net.get_net_if_addrs(i)["ipv4"]
-        logging.debug("ipv4_value is %s", ipv4_value)
-        if ipv4_value != []:
-            ip_addr = ipv4_value[0]
-            break
-    if ip_addr is not None:
-        logging.info("ipv4 address is %s", ip_addr)
-    else:
-        raise exceptions.TestFail("Fail to get ip address")
-    return ip_addr
```

Comment:
Still have other case using this func.
./libvirt/tests/src/virsh_cmd/volume/virsh_volume.py:                    ip_addr = utlv.get_host_ipv4_addr()

Reply:
why remove it as it can get ip without giving ifname and usable?

Reply:
> Still have other case using this func.
> ./libvirt/tests/src/virsh_cmd/volume/virsh_volume.py: ip_addr = utlv.get_host_ipv4_addr()

It's covered by https://github.com/autotest/tp-libvirt/pull/2281


> why remove it as it can get ip without giving ifname and usable?
It's similar with  get_host_ip_address from functionality in utils_net.py, and it's used by virsh_volume.py in tp-libvirt and original setup_or_cleanup_gluster in avocado-vt.
The func get_host_ip_address can also get ip address without ifname specified -  when it's None, it will get default interface for now.

Reply:
ok, thanks